README.md

```diff
@@ -91,7 +107,9 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo){
+  console.log("I'm gonna "+ thingToDo + " this weekend. Wanna come?");
+}
```

Comment:
or pass in a function - https://github.com/ga-wdi-quizzes/week-04/blame/6111775b35480b3fd9021976459e0a4402981ca0/README.md#L108-L113